src/molecules/formfields/shared/formfields.module.scss

```diff
@@ -112,5 +72,5 @@ $focused-border-color: color('brand-3');
 }
 
 .hasError {
-  border: 1px solid $field-error-color;
+  border: 1px solid color('accent-2');
```

Comment:
Since this is a shared color / standard for the component, I think it makes sense to enforce the consistency via a variable, no?